### ISSUE TITLE
Raise duplicate sample id problem with htslib

### ISSUE THREAD

**Issue by jeromekelleher** [ga4gh/server#521]
The  `testInstantiation` in `tests.unit.test_faultydata.TestDuplicateCallSetId` is currently disabled because it provokes an error that results in htslib aborting. This is a bug in htslib. We should:

(a) track down where in htslib this happens, and confirm the problem still exists; and 
(b) open an issue with htslib to get it fixed.

**Comment by david4096** [comment]
Verified by turning back on the test. `htslib` @jmarshall crashed with:

The VCF in question:  https://github.com/ga4gh/server/tree/master/tests/faultydata/variants/duplicated_sampleid

```
(env)➜  server-1 git:(1253_codecov) ✗ nosetests tests.unit.test_faultydata
[E::bcf_hdr_add_sample] Duplicated sample name 'S1'
[1]    39488 abort      nosetests tests.unit.test_faultydata
```

**Comment by jmarshall** [comment]
Yes, that's because it considers this malformed `#CHROM` header line terrifying enough that it prints the error message and aborts.  Did you ever file an htslib issue about this?  Then we wouldn't forget about it and would probably calm `bcf_hdr_add_sample()` down.

**Comment by david4096** [comment]
Issue opened, we'll leave this here to help track progress on the htslib side.
